qcfractal/interface/models/records.py

```diff
@@ -277,29 +278,29 @@ class ResultRecord(RecordBase):
     # Version data
     version: int = Schema(
         1,
-        description="Version of the ResultRecord Model which this data was created with"
+        description="Version of the ResultRecord Model which this data was created with."
     )
     procedure: constr(strip_whitespace=True, regex="single") = Schema(
         "single",
-        description='Fixed as "single" because this is single quantum chemistry result.'
+        description='Procedure is fixed as "single" because this is single quantum chemistry result.'
     )
 
     # Input data
     driver: DriverEnum = Schema(
         ...,
-        description="What type of calculation this result is tied to."
+        description="The type of calculation this result references."
     )
     method: str = Schema(
         ...,
-        description="What quantum chemistry method the calculations was computed with."
+        description="The quantum chemistry method the driver runs with."
     )
     molecule: ObjectId = Schema(
         ...,
-        description="The ID of the molecule in the Database which the calculation was performed on."
+        description="The ID of the molecule in the Database which the result is computed on."
     )
     basis: Optional[str] = Schema(
         None,
-        description="The basis set which the calculation was performed with."
+        description="The basis set which the calculation is performed with."
```

Comment:
"The basis set of the calculation (e.g., "sto-3g", "cc-pVDZ", ...)". I think this language is in one of the `common_models`.